sqlglot/expressions.py

```diff
@@ -3961,6 +3961,10 @@ class Between(Predicate):
 class Bracket(Condition):
     arg_types = {"this": True, "expressions": True}
 
+    @property
+    def output_name(self):
+        return self.expressions[0].this
+
```

Comment:
On second thought, I'm not sure if this correct.. What if you have a `Bracket` like `a.b[0]`? Or what if you have multiple expressions under the `Bracket`? Just saw that the return type hint was omitted as well; we require `output_name` to return a `str` but `expressions[0].this` isn't necessarily a string value (if we were to go with this approach, perhaps something like `self.expressions[0].name` would be more appropriate).

@tobymao what do you think?

Reply:
you can check if this is a literal or a var or something?

Reply:
Yeah, makes sense. An alternative solution could be:

- Check if there's just a single one or more expressions
- If there's only one, output its `name` only if it's a var / string / identifier

Reply:
If we agree on this approach I can take this PR to the finish line.

Reply:
just use the output name of the argument

Reply:
https://github.com/tobymao/sqlglot/commit/a81dd14a6de1a50438eae64c2dd20e4841c29572